lib/rex/post/meterpreter/packet.rb

```diff
@@ -314,6 +314,30 @@ def initialize(type, value = nil, compress=false)
     end
   end
 
+  def _tlv_type_string(type)
+    # Always include Meterpreter and Stdapi
+    available_modules = [
+      ::Rex::Post::Meterpreter,
+      ::Rex::Post::Meterpreter::Extensions::Stdapi
+    ]
+    defined_extensions = ::Rex::Post::Meterpreter::ExtensionMapper.get_extension_modules(::Rex::Post::Meterpreter::Extensions.constants)
+    available_modules.push defined_extensions
+    available_modules.flatten!.uniq!
+
+    available_modules.each do |clazz|
+      clazz.constants.each do |const|
```

Comment:
Might be good to cache this 🤔

Reply:
Or just compute the entirely available options upfront? I can't imagine it would be too large 🤔 

Would still need to fallback to a slow search if a value isn't found of course, just incase there was a glitch in the matrix